src/main/java/de/unistuttgart/informatik/fius/icge/course/Presets.java

```diff
@@ -28,4 +28,31 @@ public static Editor cage(int width, int height) {
 
         return ed;
     }
+    
+    /**
+     * This function generates an Editor with a array defined map with coins.
+     * A -1 in a Cell indicates a wall
+     * A 0 indicates nothing
+     * And 1 upwords indicates the number of coins in that field.
+     *  
+     * @param map the array containing a predefined map
+     * @return returns an Editor to add further work on a map
+     */
+    public static Editor loadFromArray(int[][] map) {

```

Comment:
if this function is used to generate an enclosing wall the upper left wall will have the coordinates (0,0). This is inconsistent with the cage preset and may lead to misunderstandings in the tasks.

Possible solution:
extra parameter `boolean withCage` that generates a cage around the whole array content.